workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -360,7 +360,11 @@ def process_frames_for_key(key: str,
             # Count how many frames are in each tech so we can preallocate
             # the matrices in both directions.
             if not frame['unsmashable']:
-                all_gene_identifiers = all_gene_identifiers.union(frame['dataframe'].index)
+                for gene_id in frame['dataframe'].index:
+                    if gene_id in gene_identifier_counts:
```

Comment:
Don't you mean if it's not in gene_identifier_counts?

Reply:
Haha I just had a test complain about this.

Reply:
Glad I've still got some eyes 👀